src/libs/blueprint/conduit_blueprint_mesh.cpp

```diff
@@ -1043,6 +1043,101 @@ convert_coordset_to_explicit(const std::string &base_type,
     }
 }
 
+//-------------------------------------------------------------------------
+void
+copy_node_rename_first_child(const conduit::Node & from,
+                             conduit::Node & to,
+                             const std::string & key,
+                             const std::string & new_name,
+                             const std::string & new_sibling)
+{
+    to[key].set(from[key]);
+    conduit::Node & n = to[key];
+    const std::string from_name = n.child(0).name();
+    n.rename_child(from_name, new_name);
+    n[new_sibling].set(n[new_name]);
+}
+
+//-------------------------------------------------------------------------
+void
+copy_node_substitute_sibling(const conduit::Node & from,
+                             conduit::Node & to,
+                             const std::string & key,
+                             const std::string & name,
+                             const std::string & data_sibling)
+{
+    to[key].set(from[key]);
+    conduit::Node & n = to[key];
+    n.remove_child(name);
+    n.rename_child(data_sibling, name);
+}
+
+//-------------------------------------------------------------------------
+void
+convert_oneD_coordset_to_strip(const conduit::Node &coordset,
+                               conduit::Node &dest)
+{
+    dest.reset();
+    std::string coord_type = coordset["type"].as_string();
+    dest["type"].set(coord_type);
+
+    if (coord_type == "uniform")
+    {
+        copy_node_rename_first_child(coordset, dest, "dims", "j", "i");
+        dest["dims/i"].set(1);
```

Comment:
```
copy_node_rename_first_child(coordset, dest, "dims", "j", "i");
```
This will copy `coordset["dims"]` into `dest["dims"]`.
Then rename `dest["dims"].child(0)` to be named `j` or `dest["dims/j"]`
Then copy `dest["dims/j"]` so it is also `dest["dims/i"]`
And then you reset:
`dest["dims/i"].set(1);`

Why have the sibling copy if we are going to reset it in these cases?